### PR TITLE
Spymaster Turn

### PR DESCRIPTION
**Pull request by AltiV**
## Proposed Changes

  - Added hint box + word number to board state

### CODE REVIEW

File: client/src/components/Board.js [hunk 1]
```diff
@@ -23,39 +34,59 @@ const useStyles = makeStyles((theme) => ({
...
+};
 
-const isSpy = role => role === "blue spy" || role === "red spy";
+const isSpy = (role) => role === "blue spy" || role === "red spy";
 
 const api = {
-  "nextMove": {
-    url: id => `/game/${id}/next-move`,
+  nextMove: {
+    url: (id) => `/game/${id}/next-move`,
```
Comment (sina-jamshidi) [inline client/src/components/Board.js:80]: I think you might have different prettier settings than the rest of the team. We can double check this in standup

File: client/src/components/Board.js [hunk 2]
```diff
@@ -64,109 +95,218 @@ const api = {
...
+
+  // State variables
+  const [spyHint, setSpyHint] = useState("");
+  const [spyHintNumber, setSpyHintNumber] = useState(1);
+  const [endGameModalOpen, setEndGameModalOpen] = useState(false);
+
+  // State change handlers
+  const handleSpyHintChange = (event) => {
+    // Spy can only submit one "word", with no spaces
+    setSpyHint(event.target.value.split(" ").join(""));
```
Comment (sina-jamshidi) [inline client/src/components/Board.js:122]: Nice!

**Review by sina-jamshidi — APPROVED** [review]
Looks good! There's some overlap with Travis' code, so merge with his branch first

**Comment by AltiV** [comment]
Superseded by #52.

**Pull request closed without merging**